### ISSUE TITLE
Plugin does not support Flutter 2.0 localization changes...

### ISSUE THREAD

**Issue by patniemeyer** [localizely/flutter-intl-intellij#66]
Localizations now use flutter:generate and an l10n.yaml file to describe the file locations, with a new default  naming convention `app_xx.arb`.  The plugin currently keeps trying to recreate the old `intl_xx.arb` template file, which causes build problems.

Wondering if you plan to support the new stuff.  Thanks.

**Comment by aleksakrstic** [comment]
Hi @patniemeyer 
For now, you should use either Flutter Intl IDE plugin or gen_l10n from Flutter 2.
We will see now it will align in upcoming time.

Can you share what features you try to benefit from using both?

**Comment by patniemeyer** [comment]
The plugin supports the string refactoring (extract to ARB file).   Is there another way to get this functionality for 2.0?  Thanks.

**Comment by aleksakrstic** [comment]
Hi @patniemeyer 
Thanks for the info.

You can use this plugin with Flutter 2. Just follow config steps from https://plugins.jetbrains.com/plugin/13666-flutter-intl.
Just you shouldn't mix this plugin with `l10n.yaml` approach at the same time.

**Comment by selankon** [comment]
Hi, is there the possibility of generate the code from command line like `gen_l10n`? And we can install the plugin from `pub`? I'm asking this to check how generate locales without IDE, as `gen_l10n`.

I'm also interested about the compatibility from this plugin with  `gen_l10n`.

**Comment by aleksakrstic** [comment]
Hi @selankon 
Yes, you can use `intl_utils` as CLI tool: https://pub.dev/packages/intl_utils

**Comment by patniemeyer** [comment]
> Hi @patniemeyer Thanks for the info.
> 
> You can use this plugin with Flutter 2. Just follow config steps from https://plugins.jetbrains.com/plugin/13666-flutter-intl. Just you shouldn't mix this plugin with `l10n.yaml` approach at the same time.

Perhaps I'm missing something obvious but the plugin only seems to understand the Flutter 1.x intl ARB file naming convention.  The only way that I can see to use the plugin with Flutter 2 is to create a symbolic link from `lib/l10n/intl_en.arb` to `app_en.arb`, enable the plugin, do the extractions, then disable the plugin and remove the link...  And this is what I have been doing but it's tedious.  It would be nice if the plugin would just recognize the flutter 2 naming convention and do the extraction to the right file.  Thanks.

**Comment by aleksakrstic** [comment]
Hi @patniemeyer 
I agree. The plan is to make filename pattern configurable as well, like the `arb_dir` is configurable.

In the meantime, have you tried with configuring the following inside `l10n.yaml`?

    template-arb-file: intl_en.arb

**Comment by patniemeyer** [comment]
Yes, that is a workaround.  I was just trying to follow the new Flutter conventions to avoid conflicts between the two.  I think it would be ideal if localizely recognizes the l10n.yaml and (if it exists) adopts the new naming convention and turns off its own file generation in favor of the built-in.   Or maybe you should split out the string extraction functionality into its own plugin?  Thanks.

**Comment by ipcjs** [comment]
I modified and repackaged the vscode version of this plugin to support setting filenames. You can try it out.😅

https://github.com/ipcjs/flutter-intl-vscode/releases/latest
